spec.html

```diff
@@ -19425,10 +19426,9 @@ <h1>Runtime Semantics: ArgumentListEvaluation ( ): either a normal completion co
           1. Let _spreadObj_ be ? GetValue(_spreadRef_).
           1. Let _iteratorRecord_ be ? GetIterator(_spreadObj_, ~sync~).
           1. Repeat,
-            1. Let _next_ be ? IteratorStep(_iteratorRecord_).
-            1. If _next_ is *false*, return _list_.
-            1. Let _nextArg_ be ? IteratorValue(_next_).
-            1. Append _nextArg_ to _list_.
+            1. Let _next_ be ? IteratorStepValue(_iteratorRecord_).
+            1. If _next_ is ~done~, return _list_.
+            1. Append _next_ to _list_.
```

Comment:
should all of these have an assertion that `next` is an ECMAScript language value?

Reply:
Eh, I don't think it's necessary. It's not hard to tell given the type of IteratorStepValue.